resolver.go

```diff
@@ -36,12 +36,18 @@ const (
 	ptrIPv4domain = ".in-addr.arpa."
 	ptrIPv6domain = ".ip6.arpa."
 	respTTL       = 1800
+	maxExtDNS     = 3 //max number of external servers to try
 )
 
+type extDNSEntry struct {
```

Comment:
I want to do the parsing once and set the v6 flag when the extDNS is populated rather than doing it on each DNS query handling which could be 100s of times. 

> I would just check in NameServer() with isIPv6 := net.ParseIP().to4() == nil in the loop.

Are you referring to ServeDNS() function ? That is the one handling the query where the v6 flag is checked.